src/gemm.c

```diff
@@ -318,16 +318,6 @@ void gemm_nn_custom_bin_mean_transposed(int M, int N, int K, float ALPHA_UNUSED,
 
 //----------------------------
 
-// is not used
-void transpose_32x32_bits_my(uint32_t *A, uint32_t *B, int lda, int ldb)
```

Comment:
Please, leave this function, later may be I will use it (at least for debugging).

Reply:
uncommented